extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -217,7 +218,7 @@ static public function getDomains( $city_id, $master = false ) {
 				}
 			}
 
-			$dbr = ( $master ) ? self::db( DB_MASTER ) : self::db( DB_SLAVE );
+			$dbr = ( $master ) ? static::db( DB_MASTER ) : static::db( DB_SLAVE );
 			$oRes = $dbr->select(
 				[ "city_domains" ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "city_domains" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)